File: src/main/java/com/saybetter/domain/member/exception/MemberException.java
package com.saybetter.domain.member.exception;

import com.saybetter.global.common.code.BaseErrorCode;
import com.saybetter.global.common.exception.GeneralException;

public class MemberException extends GeneralException {

	public MemberException(BaseErrorCode code) {
		super(code);
	}

}

File: core/core-api/src/main/java/io/say/better/domain/member/exception/MemberException.kt
package io.say.better.domain.member.exception;

import io.say.better.global.common.code.BaseErrorCode;
import io.say.better.global.common.exception.GeneralException;

public class MemberException extends GeneralException {

	public MemberException(BaseErrorCode code) {
		super(code);
	}

}
